data/controller.script

```diff
@@ -0,0 +1,44 @@
+SET DATABASE UNIQUE NAME HSQLDB8E5DD0FA30
```

Comment:
I believe this is a generated file; we don't want it to be included in Git.
Maybe consider adding it to `.gitignore`.